meta-sokol-flex-staging/dynamic-layers/virtualization-layer/recipes-containers/docker-compose/python3-docker-compose_%.bbappend

```diff
@@ -2,4 +2,4 @@
 # SPDX-License-Identifier: MIT
 # ---------------------------------------------------------------------------------------------------------------------
 
-RDEPENDS:${PN} += "${PYTHON_PN}-distutils"
+RDEPENDS:${PN}:feature-sokol-flex-staging += "${PYTHON_PN}-distutils"
```

Comment:
Same issue as finch, append is being converted to appending to an override which replaces the original rather than appending to it.